### ISSUE TITLE
Can't run 0.9.2 on 10.10.5

### ISSUE THREAD

**Issue by GitTheHellOutaHere** [w0lfschild/cDock#50]
cDock 8.2 was working fine until two hours ago.

That's when I downloaded and installed the new and oddly numbered 0.9.2 and have been fighting with it up until 10 minutes ago when I re-installed 8.2 and it now works again.
The app ran, it just wouldn't apply any themes no matter what I did.

And yes, I rebooted...twice.

I completely removed all traces of cDock 0.9.2 (except ~/Application Support/cDock/themes) and SIMBL, then reinstalled cDock 8.2 and SIMBL 0.9.9.

It seems v0.9.2 is not ready for release.

**Comment by GitTheHellOutaHere** [comment]
After a reboot the theme vanished and I couldn't get cDock 8.2 to work again.

While searching for answers I found cDock 0.9.8 so I decided to give it a whirl.
I installed it and let it install SIMBL (which I'd removed).

Now cDock 0.9.8 works half-assed. The first three 3D themes are applied-at least there's a shelf and I'm able to change the indicator light color. I'm not able to change the indicator size nor icon label background color.

When I switch to Concrete Light, which is the theme I used to have (and the one I want now), the Dock becomes transparent and I'm still able to change the indicator color, but that's about all.

Editor's Choice applies  a blue background all the way across the screen, with yellow indicators that are wider than normal. I'm guessing that's correct for that theme, although IMO the background should only be as wide as my Dock setting.

Hello World is applied correctly but no changes can be made.

But the vast majority of themes, including Transparent, cause the Dock to revert to its original fugly appearance.

**Comment by w0lfschild** [comment]
Yeah cDock 2 uses a different theme 'api' (I suppose is the correct term) so themes from the original cDock will not work in cDock 2. You can definitely remake them though.

**Comment by GitTheHellOutaHere** [comment]
Not exactly a timely response, but it's nice to know the problem isn't on my end.
I tried again recently and just got frustrated again.

It doesn't look too complicated to copy one of the plist files and make changes, but some also require a background image. More than I want to attempt, but now I’m unsatisfied with my Dock again.

Even making changes to the default themes in the Theming tab usually doesn’t change them.
For example I like the wider open app indicator that one theme has, so I moved the Indicator Width to the max for other themes but they remained a small dot.

    Thanks for the reply!
